README.md

```diff
@@ -6,12 +6,8 @@ Currently we provide the following images which are all based upon the [ubi-mini
 base image:
 
 - `infinispan/server` - Infinispan is executed using the Java 11 openjdk JVM
```

Comment:
Maybe we should adjust the version :)

Reply:
Good point. I have made it more generic so that we don't refer to a specific version, that way it won't become obsolete in the near future when 25 is released.